front-end/src/index.js

```diff
@@ -6,7 +6,30 @@ import App from './components/App';
 import reducer from './reducers';
 import style from './scss/index.scss';
 
-const store = createStore(reducer);
+const store = createStore(reducer, {
+    teams: [],
+    teamMembers: [
```

Comment:
I tried to use the mock JSON files, but in order to require them, I would need to pull in a json-loader library, and I didn't want to do that since pulling in those files directly is only temporary.